cspell.yaml

```diff
@@ -0,0 +1,70 @@
+version: "0.2"
+language: en
+allowCompoundWords: true
+dictionaries:
+  - node
+  - typescript
+words:
+  - autorest
+  - azsdkengsys
+  - azurecr
+  - blockful
+  - blockless
+  - cadl
+  - Contoso
+  - CRUDL
+  - dogfood
+  - eastus
+  - esbuild
+  - globby
+  - inmemory
+  - instanceid
+  - intrinsics
+  - jsyaml
+  - msbuild
+  - MSRC
+  - munge
+  - mylib
+  - nostdlib
+  - oapi
+  - oneof
+  - onig
+  - onigasm
+  - onwarn
+  - openapi
+  - openapiv
+  - picocolors
+  - proto
+  - protobuf
+  - protoc
+  - regen
+  - rpaas
+  - rushx
+  - safeint
+  - strs
+  - TSES
+  - Uhoh
```

Comment:
Where do we have this one? LOL.

Reply:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/1031227/189179547-251ed678-31e8-40be-9018-742a548467b0.png">

could rename, just found that one funny to keep :)